lib/modules/manager/mint/__fixtures__/Mintfile

```diff
@@ -0,0 +1,3 @@
+SwiftGen/SwiftGen@6.6.1
+yonaskolb/xcodegen@2.30.0
+realm/SwiftLint@0.48.0
```

Comment:
I'm curious, why is this testcase not in the fixture itself? It seem like the fixture should be the place to put all your testcases?

```
realm/SwiftLint @ 0.48.0
```

I see you test this line in the `extract.spec.ts` file though:

```ts
const complexMintFileContent = `
SwiftGen/SwiftGen@6.6.1
yonaskolb/xcodegen
realm/SwiftLint @ 0.48.0`;
```

Reply:
Thanks! 😄 I added all cases including comment outs!
https://github.com/renovatebot/renovate/pull/17351/commits/b8482afb46811642d60e3feb7a78b91ef6723537